package.json

```diff
@@ -9,7 +9,7 @@
     "type": "module",
     "scripts": {
         "gulp": "gulp --cwd gulp",
-        "lint": "(eslint . && tsc && tsc -p src) || (tsc && tsc -p src) || tsc -p src",
+        "lint": "eslint .",

```

Comment:
just to confirm, this is because when files are converted to TS, the tslint rules will cover the useful stuff `tsc` reports, right?

Reply:
Yes but not only: right now since the codebase is mostly JS, calling tsc separately for each "project" both takes time and pollutes the output. As we're not using lint tasks in CI or anything like that, tsc can be skipped.

Reply:
fair enough, and even without the `noUnusedLocals` it's still a lot of noise